catalog/spatial/geocoding/spatial-geocoding-admin-module/src/main/java/org/codice/ddf/spatial/admin/module/service/Geocoding.java

```diff
@@ -67,10 +67,7 @@ private void registerMbean() {
         }
       } catch (Exception e) {
         // objectName is not always non-null because new ObjectName(...) can throw an exception
-        LOGGER.info(
-            "Could not register MBean [{}].",
-            objectName != null ? objectName.toString() : Geocoding.class.getName(),
-            e);
+        LOGGER.info("Could not register MBean [{}].", objectName.toString(), e);
```

Comment:
Is the comment still accurate? Also the toString() should be removed from the objectName

Reply:
@clockard I'm not entirely sure. The Exception seems to be looking for more issues than `objectName` being null, so I would guess the underlying assertion that it can't register MBean is still true.

Reply:
Maybe you can re-write the comment by saying objectName cannot be null since mBeanServer is not null